src/Bootstrapper/Controllers/v1/BrandsController.cs

```diff
@@ -48,5 +48,20 @@ public async Task<IActionResult> DeleteAsync(Guid id)
             var brandId = await _service.DeleteBrandAsync(id);
             return Ok(brandId);
         }
+
+        [HttpPost("generate-random")]
+        public async Task<IActionResult> GenerateRandomAsync(GenerateRandomBrandRequest request)
```

Comment:
What are these methods for?

Reply:
They are the example methods to call the Jobs, the first one creates random seeds of brands and the second one deletes them.

Reply:
develop several hangfire usage examples with the most functionality as example (logs, progressbar, Job types (Enqueue, Schedule) ... etc)